src/SarifCli.FunctionalTests/TestData/DoNotUseFriendlyNameAsRuleId/RuleIdDoesNotMatchFriendlyName_Expected.sarif

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
The validator produces this output file for the log file that does _not_ violate the rule. There are no `result`s.